### DISCUSSION TITLE
Support for Google icons

### DISCUSSION THREAD

**Discussion by SjogrenDev** [GitbookIO/community#292] [Feature Requests]
It would be amazing to have support for https://fonts.google.com/icons. The current emojis are very limited.

**Comment by addisonschultz** [comment]
Hello @SjogrenDev ! We've just shipped this feature into GitBook, and is now available for you to add! 

Here's the docs: https://docs.gitbook.com/content-editor/editor/content-structure/content-in-a-space#add-an-icon-or-emoji-to-your-page

You can read more about adding icons and other updates we've made here: https://github.com/orgs/GitbookIO/discussions/675

**Reply by SjogrenDev** [reply]
Hi @addisonschultz Thank you for adding the feature! Please add an option to customize the color of the icons. So, the icons can match the rest of the color scheme.

**Reply by addisonschultz** [reply]
Hi! Right now they will inherit the primary color, when selected
